Directory.Build.props

```diff
@@ -38,9 +39,9 @@
     <Compile Include="**/*.cs" Exclude="$(DefaultItemExcludes)" />
   </ItemGroup>
 
-  <Target Name="DisableAnalyzers" BeforeTargets="CoreCompile" Condition="'$(Configuration)'=='Release'">
+  <Target Name="DisableAnalyzers" BeforeTargets="CoreCompile" Condition="'$(Configuration)'=='Release' Or '$(Configuration)'=='Debug'">
     <!-- Disable code style analysis on Release builds to improve compile-time performance -->
-    <ItemGroup Condition="'$(Configuration)'=='Release'">
+    <ItemGroup Condition="'$(Configuration)'=='Release' Or '$(Configuration)'=='Debug'">
```

Comment:
`'$(Configuration)'!='StyleCop'`?

Also, does `StyleCop` behave like `Debug` wrt everything that isn't stylecop, or like `Release`? IMO it would be good if the name could something be like `Debug-Strict` to make it clearer how it behaves.